Tests/EmailValidator/Validation/MailboxCheckValidationTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Egulias\Tests\EmailValidator\Validation;
+
+use Egulias\EmailValidator\EmailLexer;
+use Egulias\EmailValidator\Validation\Error\IllegalMailbox;
+use Egulias\EmailValidator\Validation\MailboxCheckValidation;
+use Egulias\EmailValidator\Warning\NoDNSMXRecord;
+use PHPUnit\Framework\TestCase;
+
+class MailboxCheckValidationTest extends TestCase
+{
+    public function testValidMailbox()
+    {
+        $validation = new MailboxCheckValidation();
+        $this->assertTrue($validation->isValid('no-reply@gmail.com', new EmailLexer()));
```

Comment:
This test is also failing and I'd argue is quite fragile, as it depends on an external resource. Also not sure you can reach gmail on port 25 :).
You could use/implement one for the test (from a quick search, you could use https://github.com/axllent/fake-smtp).

Reply:
fix tests for use mock